services/src/main/java/org/fao/geonet/api/records/MetadataSampleApi.java

```diff
@@ -267,6 +267,20 @@ SimpleMetadataProcessingReport addSamples(
                             isTemplate = templateName.startsWith(templateOfSubTemplatePrefix) ?
                                 "t" : "s";
                         }
+                        if (isTemplate.equals("s")) {
+                            // subtemplates loaded here can have a title or uuid attribute
+                            String tryUuid = xml.getAttributeValue("uuid");
+                            if (tryUuid != null && tryUuid.length() > 0) uuid = tryUuid;
```

Comment:
You could use the same as https://github.com/geonetwork/core-geonetwork/blob/master/services/src/main/java/org/fao/geonet/api/records/MetadataInsertDeleteApi.java#L797-L804

Reply:
Yep - thanks...